### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nickhudkins**

### CODE REVIEW

**Review by voidet — APPROVED** [review]

**Review by thenameisangel — APPROVED** [review]
yas

**Review by thenameisangel — DISMISSED** [review]
yas part 2

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ZOoM
 do a even cooler thing please
+maybe a review
```
Comment (schaki) [inline README.md:3]: Disagree

Comment (nickhudkins) [reply README.md:3]: you are dead to me

**Review by schaki — DISMISSED** [review]
This is against all my beliefs

File: README.md (hunk 1 above)
Comment (derekchinn) [reply README.md:3]: Where are your tests?

**Review by derekchinn — APPROVED** [review]

**Comment by nickhudkins** [comment]
nothing works. I dont understand

**Review by thenameisangel — APPROVED** [review]
do you even github

**Pull request merged**